src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Widget Library
+sidebar_order: 1
+redirect_from:
+ - /product/dashboards/customize-dashboards/
+description: "Learn how to edit or create additional dashboards to suit your organization's needs."
+---
+
+<Note>
+
+This feature is available only if your organization is on either a Business or Trial plan.
+
+</Note>
+
+The Widget Library contains a collection of prebuilt widgets you can add to your [Custom Dashboards](/product/dashboards/customize-dashboards/). To access the Widget Library, click on "Add Widget(s)" and switch to the "Widget Library" tab on the modal that appears.
```

Comment:
To avoid having to capitalize this every time we talk about it forever, I'm going to lower case this unless we're specifically calling it out in the UI.

```suggestion
The widget library contains a collection of prebuilt widgets you can add to your [custom dashboards](/product/dashboards/customize-dashboards/). To access it, click on "Add Widget(s)" and switch to the "Widget Library" tab in the modal that appears.
```